requirements-dev.txt

```diff
@@ -1,14 +1,7 @@
-# test dependencies
-pytest>=3.10.1,<7.0.0
-responses>=0.9.0,<1.0.0
-pylint>=2.0.0,<3.0.0
-tox>=2.9.1,<4.0.0
-pytest-rerunfailures>=5.0,<10.0.0
-
-# code coverage
-coverage<5,<6.0.0
-codecov>=1.6.3,<3.0.0
+codecov>=2.1.0,<3.0.0
+coverage>=4.5.4
+pylint>=2.6.0,<3.0.0
+pytest>=6.2.1,<7.0.0
 pytest-cov>=2.2.1,<3.0.0
-
-# semantic
-bumpversion>=0.5.3,<1.0.0
```

Comment:
bumpversion is not needed in this file since we explicitly install it within the travis build script.